src/main/java/algorithms/curated170/medium/LexicographicallySmallestEquivalentStringDFS.java

```diff
@@ -17,8 +14,8 @@ public String smallestEquivalentString(String A, String B, String S) {
         parent = new ArrayList[26];
 
         for (int i = 0; i < A.length(); i++) {
-            int a = A.charAt(i) - 'a';
-            int b = B.charAt(i) - 'a';
+            int a = letterIndex(A.charAt(i));
+            int b = letterIndex(B.charAt(i));
```

Comment:
It is good to remove print statements inside the business code as it slows down the run-time.
 The future audience may misvalue the efficiency if they test without removing the print part.

Reply:
I should be more careful about not forgetting them.